### PR TITLE
25 add captions to the tcplfit2 vignette figures

### PR DESCRIPTION
**Pull request by gracezhihuizhao**

### CODE REVIEW

File: vignettes/tcplfit2-vignette.Rmd [hunk 1]
```diff
@@ -577,7 +587,7 @@ legend(1e-3, 40, legend=c("Reference Dose", "Threshold Dose", "BMD-before", "BMD
...
 ```
 
-**Figure 9**: This plot shows the BMD and confidence interval before and after "censoring". The green line and green "X" mark the BMD before "censoring" and the green column represents the confidence interval. The blue line and blue "X" mark the "censored" BMD and the blue column represents the confidence interval. The brown solid line represents the maximum concentration in the data, and the dark violet dashed line represents the threshold dose set by `bmd_up_bnd`. In this example, the BMD was "censored" to the threshold value, and the confidence interval was shifted left.
+**Figure 7**: This plot shows the BMD and confidence interval before and after "censoring". The green line and green "X" mark the BMD before "censoring" and the green column represents the confidence interval. The blue line and blue "X" mark the "censored" BMD and the blue column represents the confidence interval. The brown solid line represents the maximum concentration in the data, and the dark violet dashed line represents the threshold dose set by `bmd_up_bnd`. In this example, the BMD was "censored" to the threshold value, and the confidence interval was shifted left.
```
Comment (sedavid01) [inline vignettes/tcplfit2-vignette.Rmd:590]: Suggest looking at the comments for Figure 6 as many of those may apply here.

**Review by sedavid01 — CHANGES_REQUESTED** [review]
Overall the figure captions look good.  There are some suggestions for polishing up the language for clarity and conciseness.  Also, suggest reading through each figure caption after making suggested updates to ensure there are no typos or grammatical errors, and the description still flows well.  If there are any issues feel free to edit as necessary to make sure the figure caption is well polished.

Thank you again for your hard work on updating these!

File: vignettes/tcplfit2-vignette.Rmd [hunk 2]
```diff
@@ -127,6 +129,8 @@ their own version of this. To run this example, use the following code ...
 
 ```
 
+**Figure 2:** Each figure shows the curve-fit result of a random sample from `mc3` data. Each plot has a title that contains the chemical name and assay ID. On the top of each plot displays a summary of the fitting, including what is the winning model, AC50, top, BMD, ACC, and hit-call. The concentration-response observations are the black dots, and the winning model curve is depicted by the black curve. If BMD is not `NA`, it and its boundaries are highlighted by a green line and a green rectangle. The gray bar around the x-axis represents the estimated noise region. 
+
```
[the remark this answers is not in the document]
Comment (gracezhihuizhao) [reply vignettes/tcplfit2-vignette.Rmd:133]: @sedavid01 Sentence 5 - concRespPlot dose not have a "X" marker for BMD only a vertical line. It's something I added for figures in example 5.

File: vignettes/tcplfit2-vignette.Rmd [hunk 3]
```diff
@@ -167,6 +171,8 @@ The input data for this example contains 6 signatures for one chemical in a tran
   }
 ```
 
+**Figure 3:** These figures are plots generated from using one of the plotting options in `tcplfit2`: using the `do.plot=TRUE` argument in `concRespCore` function. Each figure plots all models used to fit one row of data in `signatures`.
+
```
[the remark this answers is not in the document]
Comment (sedavid01) [reply vignettes/tcplfit2-vignette.Rmd:175]: Suggested rewording:
"This figure provides an example of plots generated using the `tcplfit2` ... Each plot displays all model fits for one row of data in the `signatures` dataset."
(Sorry for the additional rewording... Just read through the caption and it still seemed slightly off.  Feel free to re-read after this update and make any additional tweaks.)

File: vignettes/tcplfit2-vignette.Rmd [hunk 4]
```diff
@@ -178,6 +184,8 @@ The input data for this example contains 6 signatures for one chemical in a tran
   }
 ```
 
+**Figure 4:** These figures are plots generated from using one of the plotting option in `tcplfit2`: using the `concRespPlot` function. `concRespPlot`, when takes output from `concRespCore`, plots the winning curve with additional hit-call information.
```
Comment (sedavid01) [inline vignettes/tcplfit2-vignette.Rmd:187]: See suggestion for the Figure 3 caption.

File: vignettes/tcplfit2-vignette.Rmd [hunk 5]
```diff
@@ -577,7 +587,7 @@ legend(1e-3, 40, legend=c("Reference Dose", "Threshold Dose", "BMD-before", "BMD
...
+**Figure 7**: This plot shows the estimated BMD and confidence interval before and after "censoring". The green line and the "X" mark the BMD before "censoring" along with the green shaed region representing the estimated confidence interval. The blue line and the "X" mark the "censored" BMD along with the blue shaded region representing the estimated confidence interval. The solid brown line represents the maximum concentration in the data, and the dashed dark violet line represents the threshold dose set by `bmd_up_bnd`. In this example plot, the estimated BMD and the boundaries were shifted left such that the BMD was "censored" to the threshold value (blue "X" and region).
```
Comment (sedavid01) [inline vignettes/tcplfit2-vignette.Rmd:590]: Typo sentence 2 "shaed" should be "shaded".

**Review by sedavid01 — CHANGES_REQUESTED** [review]
Overall the updates are looking good thus far.  There are still some outstanding issues that need to be addressed.  Please see the comments from this review and feel free to reach out if you have any questions/comments.

**Review by sedavid01 — APPROVED** [review]
Reviewed and polished figure captions in the tcplfit2 vignette.  Marking this PR as approved and it is ready to be merged into 'dev'.

**Comment by sedavid01** [comment]
@brown-jason, changes are merged to 'master' please verify when you get the chance and if any branch clean-up is needed please either do so or provide guidance/instructions. Thanks, -Sarah

**Pull request merged**